docs/mobile-apps/automated-testing/espresso-xcuitest/xcuitest.md

```diff
@@ -817,28 +821,29 @@ This is mandatory for Android Emulators and iOS Simulators. You can find the ava
 **Real Devices**
 
 This is optional for Real Devices. There are three options you can use to determine which version you want to use for your automated Appium, Espresso, or XCUITest tests:
+
 1. Don't provide a `platformVersion`, this will result in any available Android or iOS device, no matter the version.
 2. Provide a `platformVersion` that starts with your provided `platformVersion` string:
-    * **`12`:** matches all minors and patches for `platformVersion: "12"`. For example `12.1.0|12.1.1|12.2.0|...`
-    * **`12.1`:** matches all patches for `platformVersion: "12.1"`. For example `12.1.0|12.1.1`, it will **not** match `12.2.x|12.3.x` and higher
-    * **`12.1.1`:** matches all devices that have **this exact** platform version
+   - **`12`:** matches all minors and patches for `platformVersion: "12"`. For example `12.1.0|12.1.1|12.2.0|...`
+   - **`12.1`:** matches all patches for `platformVersion: "12.1"`. For example `12.1.0|12.1.1`, it will **not** match `12.2.x|12.3.x` and higher
+   - **`12.1.1`:** matches all devices that have **this exact** platform version
 3. In/exclude a specific version and or a range of versions by using a regular expression (regex). You don't need to provide the forward slashes (`/{your-regex}/`) as you would normally do with regex. Keep in mind that the regex needs to match the format `MAJOR.MINOR.PATCH`. The possibilities are endless, but here are just a few examples:
-    * **`^1[3-4|6].*`:** Will match `13`, `14` and `16`, but not 15, see [example](https://regex101.com/r/ExICgZ/1).
-    * **`^(?!15).*`:** Will exclude version `15` with all it's minors and patches, but will match all other versions, see [example](https://regex101.com/r/UqqYrM/1).
+   - **`^1[3-4|6].*`:** Will match `13`, `14` and `16`, but not 15, see [example](https://regex101.com/r/ExICgZ/1).
+   - **`^(?!15).*`:** Will exclude version `15` with all it's minors and patches, but will match all other versions, see [example](https://regex101.com/r/UqqYrM/1).
 
 :::note NOTE
-The stricter the  `platformVersions` is,  the smaller the pool of available devices will be and the longer you might need to wait for the available device. We recommend using only the major version or using the regex option to get the best results and an available device in the fastest way.
+The stricter the `platformVersions` is, the smaller the pool of available devices will be and the longer you might need to wait for the available device. We recommend using only the major version or using the regex option to get the best results and an available device in the fastest way.
 :::
 
 ```yaml title="Use complete version for Virtual and or Real Devices"
         platformVersion: 11.0
 ```
 
-```yaml title="Use dynamic platformVersion allocation. Real Devices Only"
+````yaml title="Use dynamic platformVersion allocation. Real Devices Only"
```

Comment:
with "embeddedLanguageFormatting": "off", should it have ignored the code blocks